### PR TITLE
Split navbar into separate component

### PR DESCRIPTION
**Pull request by nbarnabee**
I've split the navbar into a separate component file and imported it into App.vue, which is what I should have done in the first place.  (The navbar is also looking better, but it's not yet properly responsive.)

### CODE REVIEW

**Comment by nbarnabee** [comment]
I've finished the navbar styling, though I had so many problems trying to get Vuetify to do what I wanted that in the end I decided to stop fighting it (their toolbar components are clearly built with mobile phones in mind) and just write it in vanilla CSS.  

I'm pleased with the results but may take another shot at Vuetify later, if there's time.  For the moment it doesn't seem like a good use of time.

**Review by Damilare1 — APPROVED** [review]

File: src/components/Navbar.vue [hunk 1]
```diff
@@ -0,0 +1,125 @@
+import { RouterLink, RouterView } from "vue-router";
+
+export default {
+<template>
+  <header>
+    <div class="left-side">
+      <img
```
Comment (Damilare1) [inline src/components/Navbar.vue:7]: Let's use the v-img like in Toolhub for the logo and the image should also be a link to the home page. https://github.com/wikimedia/toolhub/blob/main/vue/src/App.vue#L24-L30

**Review by Damilare1 — CHANGES_REQUESTED** [review]
Thanks for working on this, Looks good to me just one small suggestion and we should be good to go on my front.

**Review by HWaruguru — APPROVED** [review]

File: src/components/Navbar.vue (hunk 1 above)
Comment (nbarnabee) [reply src/components/Navbar.vue:7]: Good catch - I've enclosed both the icon and "Toolhunt" in a router link to the home page, which mimics what happens on Toolhub.

**Review by Damilare1 — APPROVED** [review]

**Pull request merged**